docs/mivot/viewer.rst

```diff
@@ -264,13 +269,13 @@ identifiers, which have the following structure: ``model:a.b``.
    mivot_instance = mivot_viewer.dm_instance
 
    print(mivot_instance.__dict__.keys())
-   dict_keys(['dmtype', 'longitude', 'latitude', 'pmLongitude', 'pmLatitude', 'epoch', 'Coordinate_coordSys'])
+   # dict_keys(['dmtype', 'longitude', 'latitude', 'pmLongitude', 'pmLatitude', 'epoch', 'Coordinate_coordSys'])
 
    print(mivot_instance.Coordinate_coordSys.__dict__.keys())
-   dict_keys(['dmtype', 'dmid', 'dmrole', 'spaceRefFrame'])
+   # dict_keys(['dmtype', 'dmid', 'dmrole', 'spaceRefFrame'])
 
    print(mivot_instance.Coordinate_coordSys.spaceRefFrame.__dict__.keys())
-   dict_keys(['dmtype', 'value', 'unit', 'ref'])
+   # dict_keys(['dmtype', 'value', 'unit', 'ref'])
```

Comment:
why are these commented out?

Reply:
If we add these as code examples for doctest, then the copybutton sphinx extension (>>> arrows on the top right corner of the box) takes care of ripping out the print results.
Try this one: https://pyvo.readthedocs.io/en/latest/mivot/viewer.html

So, overall I would rather suggest to have these formatted with the `>>>`s included